### ISSUE TITLE
Mark Failing Builds on history page

### ISSUE THREAD

**Issue by Tolmark12** [nanobox-io/ui-ux#1]
On the history page, it'd be nice to be able to flag a build as "Bad" or "Failed"

**Comment by jedgalbraith** [comment]
Failed builds/deploys are destroyed after they fail so they won't be in the list.

**Comment by sanderson** [comment]
@jedgalbraith I had a build deploy with a bad node version. This is more of a "it still went through but I don't want others to use it" kind of feature.

**Comment by jedgalbraith** [comment]
Oh. It's more of just a user decision to not use it, even though it may have technically succeeded. I see.

**Comment by sanderson** [comment]
Yep, and if we're getting really ambitious - Custom flags with custom colors. But I don't think this page is used all that often, so that may be overkill.
